tests/nlu/data_augmentation/test_augmenter.py

```diff
@@ -0,0 +1,345 @@
+import operator
+from typing import Callable, Set, Text
+
+import pytest
+from rasa.nlu.constants import TOKENS_NAMES
+from rasa.nlu.tokenizers.tokenizer import Tokenizer
+from rasa.shared.constants import (
+    DEFAULT_RANDOM_SEED,
+    LATEST_TRAINING_DATA_FORMAT_VERSION,
+)
+from rasa.shared.nlu.training_data.message import Message
+import rasa.nlu.config
+import rasa.shared.nlu.training_data.loading
+import rasa.shared.utils.components
+import rasa.shared.utils.io
+from rasa.shared.nlu.constants import (
+    INTENT,
+    TEXT,
+    VOCABULARY,
+)
+
+
+def test_augmenter_create_tokenizer():
```

Comment:
these tests cover only ~56% of the files code. Would be great to get to > 80% (this is without the CLI test which I'd rather see as integration test)